src/main/kotlin/lotto/model/Lotto.kt

```diff
@@ -0,0 +1,65 @@
+package lotto.model
+
+import lotto.util.LottoView
+
+
+enum class InputErrorCode(val message: String) {
+    NUMBER_IN_RANGE("[Error] 1에서 45 사이의 숫자여야 합니다."),
+    NUMBERS_NOT_VALID("[Error] 당첨 번호에 빈칸 혹은 문자열이 포함되어 있습니다."),
+    REPEATED_NUMBER("[Error] 입력 값 중 중복된 숫자가 있습니다.")
+}
+class Lotto(private val numbers: List<Int>) {
+    init {
+        require(numbers.size == 6) {
+            InputErrorCode.NUMBERS_NOT_VALID.message
+        }
+        validateNumbers()
+        validateRepeat()
+    }
+
+
+    private fun validateNumbers() {
+        require(numbers.all { it in 1..45 }) {
+            InputErrorCode.NUMBER_IN_RANGE.message
+        }
+    }
+
+    private fun validateRepeat() {
+        require(numbers.distinct().size == 6) {
+            InputErrorCode.REPEATED_NUMBER.message
+        }
+    }
+
+    private fun bonusCheck(inputNumbers: List<Int>, bonus: Int) : Boolean {
```

Comment:
```suggestion
    private fun bonusCheck(inputNumbers: List<Int>, bonus: Int) : Boolean =
        inputNumbers.contains(bonus)
```

이 부분이 inputNumber에 bonus가 포함되었는지 확인하는 부분이 맞다면 위와 같이 간단하게 처리할 수 있을것 같습니다.